.github/workflows/build.yml

```diff
@@ -21,16 +21,21 @@ jobs:
     name: Build
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
     - name: Compile assets
       run: |
         composer install --no-dev -o
         npm install
         npm run build
-    - name: Commit build files
+    - name: Add and commit build files
       run: |
         git config --local user.email "bot@getpantheon.com"
         git config --local user.name "Pantheon Automation"
         git add -f dist/ vendor/
+        npm run dev
```

Comment:
should we change the npm command to something else (like `npm build`) so it doesn't look like we're running dev scripts (and to prevent ourselves from assuming it's only ever used for local dev)?

Reply:
Yeah, good idea. I'll make a task for that.